### ISSUE TITLE
Test User Deduplication Script 

### ISSUE THREAD

**Issue by chloealee** [DoSomethingArchive/legacy-website#4996]
Spin up a duplicate of prod and test script to make sure it runs correctly. 

Code available here: https://github.com/DoSomething/northstar/pull/194/files

**Comment by ghost** [comment]
Blocked by https://github.com/DoSomething/phoenix/issues/4825

**Comment by angaither** [comment]
we can do this on thor now
